src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -166,17 +169,15 @@ export default class OverflowMenu extends Component {
             description={iconDescription}
             style={{ width: '100%' }}
           />
-          {!open ? null : floatingMenu ? (
+          {!options || !floatingMenu ? (
```

Comment:
This is really challenging to parse, what is the rendering logic that we need to express here?

Reply:
@joshblack Made a bit of change to make it more idiomatic - Hope it makes sense: https://github.com/carbon-design-system/carbon-components-react/pull/465/commits/c7296d0249be6d7d6df8a8458d7e27d87f75c74c